pkg/rancher-desktop/assets/translations/en-us.yaml

```diff
@@ -4742,6 +4739,43 @@ snapshots:
     icon: icon-search
     heading: No snapshots found
     body: Click on Create Snapshot to get started
+  dialog:
+    restore:
+      header: Are you sure you want to restore this Snapshot?
+      info: 'Please be aware that the current installation, including preferences, will be replaced. If the preference window is open it will be automatically closed and all unsaved changed discarded.'
```

Comment:
```suggestion
      info: 'Restoring this snapshot will replace your current installation, including preferences. If the preference window is open, it will be automatically closed, and any unsaved changes will be discarded.'
```